rubicon_ml/repository/base.py

```diff
@@ -1212,3 +1212,64 @@ def get_tags(self, project_name, experiment_id=None, entity_identifier=None, ent
         sorted_tag_data = [json.loads(tag_data[p]) for _, p in sorted_tag_paths]
 
         return sorted_tag_data
+
+    # ---------- Comments ----------
+
+    def _get_comment_metadata_root(
```

Comment:
is there any difference between this and `_get_tag_metadata_root`? the metadata root is gonna be the same - you're just changing "tag" in the filename to "comment", which happens in the function you added below this one. if so, you can just have this function call `_get_tag_metadata_root`

Reply:
Yeah, there's no real difference. I just thought it might be good form to add a comment specific one in case we wanted to differentiate between the two more in the future, but I think it's fine to remove now.

Reply:
yeah that definitely makes sense, and I could see the comments becoming a bit more complicated. if we just do something like 
```
def _get_comment_metadata_root(args):
    return self._get_tag_metadata_root(args)
```

we can have the best of both worlds - a new function available if it needs to change and reuse of the old one